### PR TITLE
48110 Direct users to FSS for Tracings

### PR DESCRIPTION
**Pull request by Shirin14926**
Code changes

### CODE REVIEW

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Index.cshtml [hunk 1]
```diff
@@ -15,7 +15,16 @@
...
     <div class="tab-pane fade show active" id="weekly" role="tabpanel" aria-labelledby="weekly-tab">
-
+        @if (@ViewBag.IsDistributor)
+        {
+            <div id="NMInfo" class="alert alert-primary p-4" role="alert">
+                <i class="bi bi-exclamation-circle-fill" aria-hidden="true"></i>
+    <div class="alert-head">Temporary Notice</div>
+    <p>
+        Redirecting to download the Weekly tracing file due to unavailability in MSI at the moment :<br>
+         <td><a href="~/nms/tracings/Title_and_Index_2022.pdf" target="_blank">Weekly Tracings File</a></td>
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Index.cshtml:25]: Please remove <td> tag as its not required here. Element 'td' cannot be nested inside element 'p'.
Also, please correct the alignment for the whole @if block.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/ShowDailyFiles.cshtml [hunk 2]
```diff
@@ -14,12 +14,23 @@
...
 <div class="tab-content" id="nmTabContent">
     <div class="tab-pane fade show active" id="daily" role="tabpanel" aria-labelledby="daily-tab">
+          @if(@ViewBag.IsDistributor)
+    {
+        <div id="NMInfo" class="alert alert-primary p-4" role="alert">
+    <i class="bi bi-exclamation-circle-fill" aria-hidden="true"></i>
+    <div class="alert-head">Temporary Notice</div>
+    <p>
+        Redirecting to download the Daily tracing file due to unavailability in MSI at the moment : <br>
+        <td><a href="~/nms/tracings/Title_and_Index_2022.pdf" target="_blank">Daily Tracings File</a></td>
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/ShowDailyFiles.cshtml:24]: Please remove <td> tag as its not required here. Element 'td' cannot be nested inside element 'p'.
Also, please correct the alignment for the whole @if block.

**Review by usha13529 — CHANGES_REQUESTED** [review]
Minor comments to fix please.

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/ShowDailyFiles.cshtml (hunk 2 above)
Comment (Shirin14926) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/ShowDailyFiles.cshtml]: Removed and alignment made proper

**Review by usha13529 — APPROVED** [review]
Thanks for the changes @Shirin14926. Looks good to me.

**Review by barrie-cooper — APPROVED** [review]
LGTM, thanks @Shirin14926 

Will wait until after Show & Tell for merge.

**Pull request merged**